resources/eventing/charts/nats/templates/statefulset.yaml

```diff
@@ -33,7 +33,9 @@ spec:
       labels:
         {{- include "nats.selectorLabels" . | nindent 8 }}
         {{- if .Values.statefulSetPodLabels }}
-        {{ toYaml .Values.statefulSetPodLabels | indent 8 }}
+        {{- range $key, $value := .Values.statefulSetPodLabels }}
+        {{ $key }}: {{ $value | quote }}
+        {{- end }}
```

Comment:
is there no easier version to achieve the same, without manually quoting and iterating over the list of Labels?

Reply:
Done